### DISCUSSION TITLE
Improve inconsistent shortcuts when Mozilla Firefox's shortcuts are disabled

### DISCUSSION THREAD

**Discussion by artjomsR** [Floorp-Projects/Floorp#995] [Feature Request]
I wanted to customise my shortcuts in Floorp so I went to Settings -> Keyboard shortcuts and disabled FF shortcuts. I've configured most of them as I like but there's 2 inconsistencies I've found that can't be configured / changed:
- Ctrl + Tab & Ctrl + Shift + Tab cannot be unbound / changed. I'm using https://github.com/mbnuqw/sidebery and I would like to bind its `Switch to the next /prev tab` shortcuts to Ctrl (+ Shift) + Tab instead. This is because sideberry allows more control with how I navigate my tabs using these shortcuts. But as of now, Floorp captures these shortcuts instead (even if they're bound in Sideberry)
- There is no configurable shortcut to focus on an address bar
  - F6 does this by default but it's not configurable. I'd like this to be configurable so that I can remap this to `Ctrl+L`, like in native FF
  - Floorp's `Search the web` shortcut is next closest thing - it focuses on the address bar but it also puts e.g. `Google` instead of the current address and one needs to press `Esc` twice to reveal the actual address

Floorp v11.10.2, same on both Windows / Linux

Let me know if you'd prefer me to create separate issues for these instead

**Comment by CutterKnife** [comment]
Due to technical issues, keyboard shortcuts are limited.
For example, the Cmd key (equivalent to Ctrl) cannot be set as a shortcut on the Mac, for compatibility reasons.

**Reply by artjomsR** [reply]
@CutterKnife I see. But is this anything specific to these 2 shortcuts? There are many other shortcuts that are present in the original FF which use the Ctrl / Cmd key and can be rebound in Floorp (e.g. `Ctrl+T` to open a new tab, `Ctrl+W` to close a tab, etc)

Is there anything that prevents either / both of these shortcuts to appear under `Custom keyboard shortcuts` in Floorp, so that they can be rebound just like all the other custom shortcuts?

**Reply by CutterKnife** [reply]
Sure, there are settings that technically can create keyboard shortcuts but do not exist.

The number of shortcuts is gradually increasing, so some may be configurable in the future. I am not a developer so I can't say for sure.
